services/venice-server/src/main/java/com/linkedin/venice/server/VeniceServer.java

```diff
@@ -625,6 +625,8 @@ public void shutdown() throws VeniceException {
         }
       }
 
+      storageService.checkWhetherStoragePartitionsShouldBeKeptOrNot(helixParticipationService.getHelixManager());
+
```

Comment:
During the meeting, Zac mentioned that we should move this back to startup.

Please change it back to how it was previously. My bad.
`managerFuture.thenApply(manager -> {
      storageEngine.checkWhetherStoragePartitionsShouldBeKeptOrNot(manager);
      return true;
    });`

Reply:
This has been updated accordingly.